README.md

```diff
@@ -235,6 +235,19 @@ import chartmogul
 chartmogul.Transaction.create(config, uuid='inv_745df1d4-819f-48ee-873d-b5204801e021', data={})
 ```
 
+#### [SubscriptionEvents](https://dev.chartmogul.com/docs/subscription_events)
+
+```python
+import chartmogul
+chartmogul.SubscriptionEvent.all(config)
+chartmogul.SubscriptionEvent.create(config, data={})
+chartmogul.SubscriptionEvent.modify(config, id=73966836, data={
```

Comment:
Shouldn't this be `modify_with_params` instead? Also, `id` (or `external_id` and `data_source_uuid`) should be a part of the `data` payload.